main.go

```diff
@@ -15,9 +15,10 @@ import (
 	"github.com/urfave/cli"
 )
 
-// version will be populated by the Makefile, read from
-// VERSION file of the source code.
-var version = ""
+// version is normally set from the contents of VERSION file by go build's
+// -X main.version= option in the Makefile. In case it's not set, the value
+// here provides a sane default (matching the VERSION modulo +prefix).
+var version = "1.0.0-rc95+unknown"
```

Comment:
If we do it this way then we technically don't need to have `VERSION` at all (after all the idea, of `VERSION` was specifically so we didn't need to have the version string in the source code). You can just grab the version by just `grep`-ing it out.

(I also am not sure I like that the `+dev` prefix is dropped and tagged releases will have the same version as untagged commits. If we're going to put the version in the source we should just get rid of `VERSION` -- at least until we can switch to `go:embed` and go back.)

Reply:
In this case I'd rather go back to the original version of this PR -- having `unknown` as the default.